packages/relay/src/lib/services/debugService/index.ts

```diff
@@ -203,6 +244,34 @@ export class DebugService implements IDebugService {
     return { resolvedFrom, resolvedTo };
   }
 
+  /**
+   * Returns the final formatted response for opcodeLogger config.
+   * @async
+   * @param {string} transactionHash - The hash of the transaction to be debugged.
+   * @param {object} tracerConfig - The tracer config to be used.
+   * @param {boolean} tracerConfig.disableMemory - Whether to disable memory.
+   * @param {boolean} tracerConfig.disableStack - Whether to disable stack.
+   * @param {boolean} tracerConfig.disableStorage - Whether to disable storage.
+   * @param {string} requestIdPrefix - The request prefix id.
+   * @returns {Promise<object>} The formatted response.
+   */
+  async callOpcodeLogger(
+    transactionHash: string,
+    tracerConfig: { disableMemory?: boolean; disableStack?: boolean; disableStorage?: boolean },
```

Comment:
I think we should separate with a comma here?

Reply:
If you are referring to the type declaration, usually semicolons are used in those.